lib/config.js

```diff
@@ -15,15 +15,15 @@ module.exports = {
     go: 15000,
     haskell: 15000,
     sql: 14000,
-    java: 20000
+    java: 20000,
+    kotlin: 20000,
```

Comment:
A very common complain about kotlin is that it compiles slower than Java. Pehaps you could consider adding some morer time.